src/vorta/assets/UI/mainwindow.ui

```diff
@@ -159,41 +159,17 @@
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
-      <item row="1" column="1" alignment="Qt::AlignTop">
-       <widget class="QLabel" name="createProgressText">
-        <property name="sizePolicy">
-         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
-          <horstretch>0</horstretch>
-          <verstretch>0</verstretch>
-         </sizepolicy>
-        </property>
-        <property name="minimumSize">
-         <size>
-          <width>0</width>
-          <height>55</height>
-         </size>
-        </property>
-        <property name="font">
-         <font>
-          <pointsize>11</pointsize>
-         </font>
-        </property>
-        <property name="text">
-         <string/>
-        </property>
-        <property name="alignment">
-         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
-        </property>
-        <property name="wordWrap">
-         <bool>true</bool>
-        </property>
-       </widget>
-      </item>
       <item row="1" column="0" alignment="Qt::AlignTop">
        <widget class="QPushButton" name="cancelButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
+        <property name="maximumSize">
+         <size>
+          <width>150</width>
+          <height>16777215</height>
+         </size>
+        </property>
```

Comment:
You can avoid having to explicitly set widths for all of the surrounding elements if you set the horizontal policy of the new text label, exactly like the progress bar previously had
https://github.com/SergioRAgostinho/vorta/blob/5752233ab010299189ffae0845da6f5dd470df16/src/vorta/assets/UI/mainwindow.ui#L228-L233

Reply:
Good tip. Changed.